src/Symfony/Component/HttpClient/Tests/DataCollector/HttpClientDataCollectorTest.php

```diff
@@ -199,19 +298,19 @@ public function testItDoesNotFollowRedirectionsWhenGeneratingCurlCommands()
     {
         $sut = new HttpClientDataCollector();
         $sut->registerClient('http_client', $this->httpClientThatHasTracedRequests([
-                [
-                    'method' => 'GET',
-                    'url' => 'http://symfony.com/releases.json',
```

Comment:
This case don't need to be updated.

Reply:
@GromNaN  I changed it because the test was making a request to symfony.com, so I though it'd be better if it didn't rely on a 3rd party service. I can revert it if you want.

Reply:
Just for the records, this test's purpose was to check that the redirections are not followed (in this case http → https), so the returned cURL command corresponds exactly to the request.

Reply:
@Deuchnord Tnx for the info, I totally missed that. I've updated the test to go to a url that redirects.